File: GTTS/INL.BudgetsService/Models/CustomBudgetItem_Item.cs
using System;

namespace INL.BudgetsService.Models
{
    public class CustomBudgetItem_Item
    {
        public long? CustomBudgetItemID { get; set; }
        public long? TrainingEventID { get; set; }
        public bool IsIncluded { get; set; }
        public long? LocationID { get; set; }
        public int? BudgetCategoryID { get; set; }
        public int? CustomBudgetCategoryID { get; set; }
        public bool SupportsQuantity { get; set; }
        public bool SupportsPeopleCount { get; set; }
        public bool SupportsTimePeriodsCount { get; set; }
        public string Name { get; set; }
        public decimal Cost { get; set; }
        public int? Quantity { get; set; }
        public int? PeopleCount { get; set; }
        public decimal? TimePeriodsCount { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public DateTime ModifiedDate { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.BudgetsService.Models/custom-budget-item_item.ts



export class CustomBudgetItem_Item {
  
	public CustomBudgetItemID?: number;
	public TrainingEventID?: number;
	public IsIncluded: boolean = false;
	public LocationID?: number;
	public BudgetCategoryID?: number;
	public CustomBudgetCategoryID?: number;
	public SupportsQuantity: boolean = false;
	public SupportsPeopleCount: boolean = false;
	public SupportsTimePeriodsCount: boolean = false;
	public Name: string = "";
	public Cost: number = 0;
	public Quantity?: number;
	public PeopleCount?: number;
	public TimePeriodsCount?: number;
	public ModifiedByAppUserID: number = 0;
	public ModifiedDate: Date = new Date(0);
  
}


